scaleway/resource_lb_backend.go

```diff
@@ -307,7 +307,7 @@ func resourceScalewayLbBackendCreate(ctx context.Context, d *schema.ResourceData
 			HTTPSConfig:     expandLbHCHTTPS(d.Get("health_check_https")),
 		},
 		ServerIP:           expandStrings(d.Get("server_ips")),
-		SendProxyV2:        d.Get("send_proxy_v2").(bool),
+		SendProxyV2:        d.Get("send_proxy_v2").(*bool),
```

Comment:
should it be a expandBoolPtr ? Regardless, it shouldn't be in this PR.

Reply:
commit removed